R/extract.R

```diff
@@ -148,11 +170,34 @@ extract.ARIMA <- function (model, include.pvalues = FALSE, include.aic = TRUE,
   return(tr)
 }
 
+#' \code{\link{extract}} method for \code{ARIMA} objects
+#'
+#' \code{\link{extract}} method for \code{ARIMA} objects. These objects are
+#' created by the \code{\link[forecast]{ARIMA}} function in the \pkg{forecast} package.
+#'
+#' @param model A model object.
+#' @param include.pvalues Report p values in the GOF block?
+#' @param include.aic Report AIC in the GOF block?
+#' @param include.aicc Report AICC in the GOF block?
+#' @param include.bic Report BIC in the GOF block?
+#' @param include.loglik Report log likelihood in the GOF block?
```

Comment:
Again, inherit some parameters? I think it would be best to have to or three basic methods from which the others can inherit. I would vote for `lm`, `glm`, and `lme4`. I think a more general version of the description of `...` is contained in `lm`.